source/tyr/tyr/binarisation.py

```diff
@@ -171,6 +171,9 @@ def fusio2ed(self, instance_config, filename, job_id, dataset_uid):
         connection_string = make_connection_string(instance_config)
         params.append("--connection-string")
         params.append(connection_string)
+        params.append("--local_syslog")
```

Comment:
This has to be optionnal

Reply:
Yes it is, if you don't call the flag, there is no redirection. :smiley: